draft-ietf-anima-brski-prm.md

```diff
@@ -1103,589 +1172,590 @@ pledgeenrollrequesttrigger = {
+ |                  |                 |                 |            |
  |                  |          [accept device?]         |            |
- |                  |          [contact vendor]         |            |
  |                  |                 |                 |            |
  |                  |                 |<------------mTLS------------>|
  |                  |                 |--------------RVR------------>|
+ |                  |                 |                 ~            |
  |                  |                 |              [extract DomainID]
- |                  |                 |              [update audit log]
+ |                  |                 |              [update audit-log]
+ |                  |                 |                 ~            |
  |                  |                 |<-----------Voucher-----------|
- |                  |<----Voucher-----|                 |            |
+ |                  |<----Voucher''---|                 |            |
  |                  |                 |                 |            |
  ~                  ~                 ~                 ~            ~
 ~~~~
 {: #exchangesfig_uc2_3 title="Voucher issuing exchange" artwork-align="center"}
 
-The HTTP request Content-Type header field for JSON-in-JWS PVR is: `application/voucher-jws+json` (see {{tpvr}} for the content definition), as defined in {{I-D.ietf-anima-jws-voucher}}.
-
-The Registrar-Agent sets the Accept field in the request-header indicating the acceptable Content-Type for the Voucher.
+As a first step of the interaction with the domain registrar, the Registrar-Agent SHALL supply the PVR artifact(s) to the registrar via HTTP-over-TLS POST to the registrar endpoint at `/.well-known/brski/requestvoucher`.
+Note that this is the same endpoint as for BRSKI described in {{Section 5.2 of !RFC8995}}.
+The request body MUST contain one previously acquired PVR artifact as defined in {{pvr_artifact}}.
+In the request header, the Content-Type field MUST be set to `application/voucher-jws+json` and the Accept field SHOULD be set to `application/voucher-jws+json` as defined in {{I-D.ietf-anima-jws-voucher}}.
 
-The HTTP response Content-Type header field is set to `application/voucher-jws+json` as defined in {{I-D.ietf-anima-jws-voucher}} if no content negotiation is used.
+Upon receiving a PVR artifact, the registrar accepts or declines the request to join the domain.
+For this, it MUST perform pledge authorization as defined in {{Section 5.3 of RFC8995}}.
+Due to the Registrar-Agent in the middle, the registrar MUST verify in addition that
 
+* the `agent-provided-proximity-registrar-cert` field of the PVR contains a registrar LDevID certificate signed by the same domain owner as the registrar-own LDevID certificate;
+  TODO(RegAgt may change registrar or reg cert may change; allow for multi-registrar installations; many installations same cert)
```

Comment:
Hm, this is harder to understand. 
Proposal:
the `agent-provided-proximity-registrar-cert` field of the PVR contains a registrar LDevID certificate signed by the same domain owner  as the registrar LDevID certificate used to sign the RVR (this accommodates domains with multiple registrars) ;

Reply:
Done